internal/pkg/scalable/cronjobs.go

```diff
@@ -0,0 +1,87 @@
+package scalable
+
+import (
+	"context"
+	"fmt"
```

Comment:
if the suspend is nil the cronjob is not suspended right? so why then care that it doesn't have the attribute. i would just do it like this:
```suggestion
// ScaleUp upscale the resource when the downscale period ends
func (c cronJob) ScaleUp() error {
	const suspend = false

	currentSuspendIsTrue, err := c.getSuspend()
	if err != nil {
		return fmt.Errorf("failed to get current replicas for workload: %w", err)
	}
	if !currentSuspendIsTrue {
		slog.Debug("workload is already upscaled, skipping", "workload", c.GetName(), "namespace", c.GetNamespace())
		return nil
	}

	c.setSuspend(suspend)
	return nil
}
```